plugins/c8y_log_plugin/Cargo.toml

```diff
@@ -1,38 +1,42 @@
 [package]
-name = "tedge_logfile_request_plugin"
-version = "0.6.4"
+name = "c8y_log_plugin"
+version = "0.1.0"
 authors = ["thin-edge.io team <info@thin-edge.io>"]
 edition = "2021"
 rust-version = "1.58.1"
 license = "Apache-2.0"
-description = "Thin.edge.io operation plugin for Cumulocity log request"
+description = "Thin-edge device logfile retriever for Cumulocity"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [package.metadata.deb]
-pre-depends = "tedge_mapper"
+maintainer-scripts = "../../configuration/debian/c8y_log_plugin"
 assets = [
-    ["../../configuration/contrib/operations/c8y/c8y_LogfileRequest", "/etc/tedge/operations/c8y/", "644"],
-    ["target/release/tedge_logfile_request_plugin", "/usr/bin/tedge_logfile_request_plugin", "755"],
+    ["../../configuration/init/systemd/c8y-log-plugin.service", "/lib/systemd/system/c8y-log-plugin.service", "644"],
+    ["target/release/c8y_log_plugin", "/usr/bin/c8y_log_plugin", "755"],
 ]
 
 [dependencies]
 anyhow = "1.0"
-async-trait = "0.1"
 c8y_api = { path = "../../crates/core/c8y_api" }
 c8y_smartrest = { path = "../../crates/core/c8y_smartrest" }
+clap = { version = "3.0", features = ["cargo", "derive"] }
 csv = "1.1"
-futures = "0.3"
-mockall = "0.10"
-reqwest = { version = "0.11", default-features = false, features = ["json", "rustls-tls"] }
+download = { path = "../../crates/common/download" }
```

Comment:
Surprised to see this crate as a dependency. What does this log plugin "download"?